podcasts/Sharing/SharingFooterView.swift

```diff
@@ -64,8 +69,16 @@ struct SharingFooterView: View {
 
     @ViewBuilder func button(destination: ShareDestination, style: ShareImageStyle, clipUUID: String, source: AnalyticsSource) -> some View {
         Button {
-            Task.detached {
-                try await destination.share(option, style: style, clipUUID: clipUUID, source: source)
+            isExporting = true
+            shareTask = Task.detached {
+                do {
+                    try await destination.share(option, style: style, clipTime: clipTime, clipUUID: clipUUID, progress: $progress, source: source)
```

Comment:
I get 2 warnings here:
```
Non-sendable type 'Binding<Float?>' in implicitly asynchronous access to main actor-isolated property '$progress' cannot cross actor boundary
```
```
Passing argument of non-sendable type 'Binding<Float?>' into main actor-isolated context may introduce data races
```

Reply:
Marking this as `MainActor` seemed to fix this.